### PR TITLE
Whitelist WXDAI token on xdai

### PR DESCRIPTION
**Pull request by nestorbe**
Added WXDAI to the tokens list on xdai.

### CODE REVIEW

File: data/tokens.json [hunk 1]
```diff
@@ -134,6 +134,12 @@
   "xdai": {
     "baseTokenName": "xDAI",
     "genName": "xGEN",
-    "tokens": {}
+    "tokens": {
+      "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d": {
```
Comment (orenyodfat) [inline data/tokens.json:138]: please make it small letters .

Comment (nestorbe) [reply data/tokens.json]: done!

**Comment by dOrgJelli** [comment]
@orenyodfat I think that the tests will pass if they're run again, not sure the changes made here would cause the error that's occurring. Seems to be a timing inconsistency.

**Comment by orenyodfat** [comment]
@nestorbe please pr vs dev not vs master .

**Comment by nestorbe** [comment]
changed @orenyodfat !

File: src/subgraph_endpoints.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
-  "http_main": "https://api.thegraph.com/subgraphs/name/daostack/v41_4",
-  "ws_main": "wss://api.thegraph.com/subgraphs/name/daostack/v41_4",
-  "http_rinkeby": "https://api.thegraph.com/subgraphs/name/daostack/v41_4_rinkeby",
-  "ws_rinkeby": "wss://api.thegraph.com/subgraphs/name/daostack/v41_4_rinkeby",
-  "http_kovan": "https://api.thegraph.com/subgraphs/name/daostack/v41_4_kovan",
-  "ws_kovan": "wss://api.thegraph.com/subgraphs/name/daostack/v41_4_kovan",
-  "http_xdai": "https://api.thegraph.com/subgraphs/name/daostack/v41_4_xdai",
-  "ws_xdai": "wss://api.thegraph.com/subgraphs/name/daostack/v41_4_xdai",
+  "http_main": "https://api.thegraph.com/subgraphs/name/daostack/v41_8",
```
Comment (orenyodfat) [inline src/subgraph_endpoints.json:2]: is this related ?

File: src/components/Scheme/SchemeInfoPage.tsx [hunk 3]
```diff
@@ -106,7 +106,8 @@ export default class SchemeInfo extends React.Component<IProps, null> {
       (scheme.uGenericSchemeParams && scheme.uGenericSchemeParams.votingMachine) ||
       (scheme.contributionRewardParams && scheme.contributionRewardParams.votingMachine) ||
       (scheme.schemeRegistrarParams && scheme.schemeRegistrarParams.votingMachine) ||
-      (scheme.contributionRewardExtParams && scheme.contributionRewardExtParams.votingMachine)
+      (scheme.contributionRewardExtParams && scheme.contributionRewardExtParams.votingMachine) ||
```
Comment (orenyodfat) [inline src/components/Scheme/SchemeInfoPage.tsx:109]: is this related?
